packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -114,35 +149,40 @@ registerInternalPlugin('ediscovery', EDiscovery, {
         name: 'transformReportRequestEmailsToUUIDS',
         direction: 'outbound',
         fn(ctx, object) {
-          if (!object) {
-            throw Error('Invalid parameter');
-          }
-
-          if (object.emails.length <= 0) {
+          if (!object || object.emails.length <= 0) {
             return Promise.resolve();
           }
 
           const promises = [];
+          const invalidEmails = [];
           object.emails.forEach((userEmail) => {
             promises.push(
               // The people list API only supports a single email
               ctx.spark.people.list({email: userEmail})
                 .then((res) => {
+                  let uuid = '';
                   if (res.items.length <= 0) {
-                    throw Error(`Unable to find user with email ${userEmail}`);
+                    ctx.spark.logger.error(`Unable to find user with email ${userEmail}`);
+                    invalidEmails.push(userEmail);
+                  }
+                  else {
+                    // User id is base64 encoded and of the format Y2lzY29zcGFyazovL3VzL1BFT1BMRS81ZDU5Yjc5NS02ZmEyLTQ2NTQtOGVjMi03NjlkYjE1YzBkOWU
+                    const decodedId = Buffer.from(res.items[0].id, 'base64').toString();
```

Comment:
FYI there is also a base64 util in `common` but this works just fine. I wonder if we even need the `common` one because it adds complexity by bringing in another plugin.